app/channels/room_channel.rb

```diff
@@ -11,6 +11,15 @@ def unsubscribed
   def speak(data)
     # ActionCable.server.broadcast "room_channel", message: data["message"]
     #have access to current user here
-    Message.create! content: "#{current_user.username}: #{data["message"]}"
+    client_action = data["message"]
+    binding.pry
```

Comment:
be wary of committing `binding.pry` lines to the main repo. other people on your team will not appreciate having to figure out why the server's taking so long to return something :smiley: